velox/connectors/hive/HiveConnectorUtil.cpp

```diff
@@ -576,7 +576,6 @@ bool testFilters(
         partitionKey,
     const std::unordered_map<std::string, std::shared_ptr<HiveColumnHandle>>&
         partitionKeysHandle) {
-  VELOX_CHECK_EQ(partitionKey.size(), partitionKeysHandle.size());
```

Comment:
Can you remove the empty line as well?  Seems the lint is not happy with it

Reply:
It's done.